bin/check-engine.js

```diff
@@ -51,5 +51,6 @@ checker(process.argv[2]).then((result) => {
     }
     else {
         console.log('\n', result.message.text.boldUnderlineError);
+        process.exit(1);
```

Comment:
If I understand the code correctly, `result.packages.length` would be the total number of packages/engines validated, not the number that failed validation.

I originally tried to return the number of failed validations, but was having some trouble reading the code and understanding the result object.  I did this as a short putt that solved the immediate problem.